lib/ddtrace/contrib/que/tracer.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'ddtrace/contrib/analytics'
+
+module Datadog
+  module Contrib
+    module Que
+      # Tracer is a Que's server-side middleware which traces executed jobs
+      class Tracer
+        def call(job)
+          trace_options = {
+            service:   configuration[:service_name],
+            span_type: Datadog::Ext::AppTypes::WORKER
+          }
+          request_span = tracer.trace(Ext::SPAN_JOB, trace_options)
+
+          request_span.resource = job.class.name.to_s
+          request_span.set_tag(Ext::TAG_JOB_QUEUE, job.que_attrs[:queue])
+          request_span.set_tag(Ext::TAG_JOB_ID, job.que_attrs[:id])
+          request_span.set_tag(Ext::TAG_JOB_ARGS, job.que_attrs[:args])
```

Comment:
`:args` and `:data` can contain personally identifiable information, such an email address, which we normally don't want shipped to Datadog by default.

We should handle this case with conditional settings (one for `data` one for `args`), like we do for Sidekiq, for example: https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md#sidekiq

Reply:
That makes sense. Thank you for catching that too. 

Fixed now too.